### PR TITLE
Get rid of mxempty()

### PR DESCRIPTION
**Pull request by musm**
How useful is this? Seems pointless to me.

### CODE REVIEW

File: src/MATLAB.jl [hunk 1]
```diff
@@ -70,4 +71,10 @@ end
 
 @deprecate duplicate(mx::MxArray) copy(mx::MxArray)
 
+export mxempty
```
Comment (yuyichao) [inline src/MATLAB.jl]: I think you can use `@deprecate` although this is fine too...

File: src/MATLAB.jl [hunk 2]
```diff
@@ -70,4 +71,10 @@ end
...
+function mxempty() 
+    Base.depwarn("MATLAB.mxempty is deprecated, use mxarray instead.", :nfields)
```
Comment (yuyichao) [inline src/MATLAB.jl]: Except the `nfields` part.

**Comment by musm** [comment]
thanks, fixed

File: src/MATLAB.jl [hunk 3]
```diff
@@ -15,7 +15,7 @@ export is_double, is_single,
        is_numeric, is_complex, is_sparse, is_empty,
        is_logical, is_char, is_struct, is_cell
 
-export mxarray, mxempty, mxsparse, delete, duplicate,
+export mxarray, mxsparse, delete,
```
Comment (yuyichao) [inline src/MATLAB.jl]: Note that you'll get a merge conflict with the other PR here.

**Pull request merged**